beacon-chain/rpc/eth/beacon/handlers_validator.go

```diff
@@ -234,7 +260,21 @@ func (bs *Server) GetValidatorBalances(w http.ResponseWriter, r *http.Request) {
 	}
 	isFinalized := bs.FinalizationFetcher.IsFinalized(ctx, blockRoot)
 
-	rawIds := r.URL.Query()["id"]
+	var rawIds []string
+	if r.Method == http.MethodGet {
+		rawIds = r.URL.Query()["id"]
+	} else {
+		err = json.NewDecoder(r.Body).Decode(&rawIds)
+		switch {
+		case err == io.EOF:
```

Comment:
Those errors case seem not to be tested.
(Not sure about the testing policy in that case.)

Reply:
done